schema.cc

```diff
@@ -432,6 +432,21 @@ schema::schema(const schema& o)
     init_view_info_from_other(o);
 }
 
+schema::schema(const schema& s, reversed_tag)
```

Comment:
Would be good to add a comment somewhere about the meaning of "reversed", i.e., that it only applies to clustering keys. It doesn't apply to partition keys.

Reply:
It seems cleaner to have a tag as a first argument. It would also behave better if we decide to add more arguments in the future.